pkg/operator/sync.go

```diff
@@ -417,6 +425,16 @@ func (optr *Operator) waitForDaemonsetRollout(resource *appsv1.DaemonSet) error
 	})
 }
 
+func (optr *Operator) waitForControllerConfigToBeCompleted(resource *mcfgv1.ControllerConfig) error {
+	return wait.Poll(controllerConfigCompletedInterval, controllerConfigCompletedTimeout, func() (bool, error) {
+		if err := isControllerConfigCompleted(resource, optr.ccLister.Get); err != nil {
+			glog.Errorf("controllerconfig is not completed: %v", err)
```

Comment:
Digging into this logging a bit, I'm seeing a LOT of logging every second which seems.. maybe not optimal?

```E0125 23:05:09.135566       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:10.135463       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:11.135672       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:12.135570       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:13.135567       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:14.135561       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:15.135600       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:16.135575       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:17.135569       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:18.135574       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:19.135574       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
E0125 23:05:20.135568       1 sync.go:431] controllerconfig is not completed: status for ControllerConfig machine-config-controller is being reported for 0, expecting it for 1
```

Reply:
done with https://github.com/openshift/machine-config-operator/pull/339/commits/f19f48d417fedd3bd1b173df9be9bb054715fbf7 -> https://github.com/openshift/machine-config-operator/pull/339/commits/4d1fffaa3a5aa64e38ea91f6be060ee89764f9aa

Reply:
Awesome, will re-pull and verify. Thanks!

Reply:
This has been fixed. Thanks @abhinavdahiya !